Sources/XcodeProj/Project/XCBreakpointList.swift

```diff
@@ -316,6 +316,7 @@ public final class XCBreakpointList: Equatable, Writable {
             case symbolic = "Xcode.Breakpoint.SymbolicBreakpoint"
             case ideConstraintError = "Xcode.Breakpoint.IDEConstraintErrorBreakpoint"
             case ideTestFailure = "Xcode.Breakpoint.IDETestFailureBreakpoint"
+            case runtimeException = "Xcode.Breakpoint.RuntimeIssueBreakpoint"
```

Comment:
if it's not too much hassle can we keep the naming close to how Xcode names it `runtimeIssue` ?

Reply:
[5081efd](https://github.com/tuist/XcodeProj/pull/761/commits/5081efd1bf99614754bdab4ef44aa6b3cfd7d74d)